### PR TITLE
fix(openchallenges): update platforms

### PR DESCRIPTION
**Pull request by vpchung**
This goes hand-in-hand with the recent DB update (#2234), where a new platform has been added.

## Preview
**Landscape**
Screenshot 2023-10-16 at 3 57 19 PM

**Platforms section**
Screenshot 2023-10-16 at 3 57 10 PM

(Updated [Figma here](https://www.figma.com/file/kjlOBxTQpJY79hUju4LVUT/OC-Platforms?type=design&node-id=3-2&mode=design&t=EfduBwV0FckXDVIH-0))

### CODE REVIEW

**Review by tschaffter — APPROVED** [review]

File: libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h1>
     <div class="metrics-text-box">
       <h2
         *ngIf="orgCount$ | async as orgCount"
-        [countUp]="12"
+        [countUp]="13"
```
Comment (tschaffter) [inline libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html:77]: This value should be fetched from the backend.

**Review by tschaffter — CHANGES_REQUESTED** [review]
Get the

File: libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html (hunk 1 above)
Comment (vpchung) [reply libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html:77]: @rrchai can you help with this? ^

Comment (rrchai) [reply libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html:77]: sure, I have created a ticket (#2241) regarding this task in my TO-DO list later this week~

**Review by rrchai — APPROVED** [review]

File: libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html (hunk 1 above)
Comment (vpchung) [reply libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html:77]: Thank you!! I appreciate it 🙏🏼

**Review by tschaffter — APPROVED** [review]
My earlier comment will be addressed in a ticket assigned to @rrchai . This PR is now ready to be merged.

**Pull request merged**